src/storage/users.ts

```diff
@@ -5,6 +5,30 @@ import { IUser } from "@rocket.chat/apps-engine/definition/users";
 
 const assoc = new RocketChatAssociationRecord(RocketChatAssociationModel.MISC, 'users');
 
+export  async function connect_user_to_clickup_uid(read: IRead, persistence: IPersistence,uid: string, user: string): Promise<void> {
```

Comment:
just for curiosity, can you explain why some methods are been named as pascal case?

Reply:
Just to make it more readable as it serves only a specific purpose.